src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -61,4 +67,14 @@
      */
     String getName();
 
+    @Override
+    default Text toText() {
```

Comment:
My thought on this is that is a default impl to avoid braking stuff. IMO `default` hints to a default and not a fixed value. Thus plugins can override this method but have an example how the others will look and feel like. And they do so across different server impls. you could also use Mixins/Transformers to override this method with a different default.
For translatable `CatalogType`s I thought of adding a new interface `CatalogType.Translatable extends CatalogType, Translatable` overwriting this method accordingly or doing so in SpongeCommon `SpongeCatalogType.Translatable`.

For the matter of being too specific... we could also push it to a factory or using new color/style/format consts. 

And If your your plugin needs to show the `CatalogType` instance in a different way the javadocs explicitly allow using a different style although using the default style is recommended.

@gabizou After this argumemtation do you stick to your opinion that this should be in impls only or do you just want it altered a little? Others?